kong/clustering/data_plane.lua

```diff
@@ -175,55 +170,44 @@ function _M:communicate(premature)
   --                 and is also responsible for handling timeout detection
 
   local ping_immediately
```

Comment:
The `ping_immediately` variable does not seem to be used anymore.

Reply:
No, `ping_immediately` is used in `write_thread`.

If this PR is merged I will create a new PR which tries to remove it.